src/hooks/useWithdrawLiquidityHandler.tsx

```diff
@@ -22,342 +18,201 @@ export const useWithdrawLiquidityHandler = () => {
   const [token0, setToken0] = useState<Token>(Tokens[0])
   const [token1, setToken1] = useState<Token>(Tokens[1])
   const [enableRemove, setEnableRemove] = useState<Boolean>(false)
-  const [uniPrice, setUniPrice] = useState<string>('0')
-  const [liqInTokB, setLiqInTokB] = useState<number>(0)
-
-  let DEPOSIT_POOL_ADDRESS = '0x3eFadc5E507bbdA54dDb4C290cc3058DA8163152'
+  const [gammaPoolContract, setGammaPoolContract] = useState<Contract | null>(null)
+  const [enableApprove, setEnableApprove] = useState<Boolean>(false)
 
   async function changeSliderPercentage(percentage: number) {
+    if (percentage > 0) {
```

Comment:
does this work?
setEnableApprove(percentage > 0)

Reply:
yeah I think it should work.

Reply:
Done